library/tinkerpop/src/main/java/uk/gov/gchq/gaffer/tinkerpop/GafferPopGraph.java

```diff
@@ -457,6 +459,37 @@ public Iterator<GafferPopVertex> verticesWithView(final Iterable<Object> ids, fi
         return verticesWithSeedsAndView(getElementSeeds(ids), view);
     }
 
+    /**
+     * This performs getRelatedEntities operation on Gaffer.
```

Comment:
is this correct

Reply:
@tb06904 yeah I left this as a reminder for me to ask about these comments. I just took this javadoc from the other vertices methods but I don't think it makes sense for any of them?